src/main/java/com/codeday/productivity/controller/UserController.java

```diff
@@ -0,0 +1,66 @@
+package com.codeday.productivity.controller;
+
+import com.codeday.productivity.entity.User;
+import com.codeday.productivity.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1")
+public class UserController {
```

Comment:
Also, you should start adding unit tests for these functionalities.  In general, you should think about following test driven development approach so as to write unit tests as the same time as feature code or before writing feature code.